it/uk/gov/hmrc/agentsubscriptionfrontend/stubs/AuthStub.scala

```diff
@@ -71,6 +71,42 @@ object AuthStub {
     sessionKeysForMockAuth(user)
   }
 
+  def isSubscribedToMtd(user: SampleUser): Unit = {
+    stubFor(get(urlEqualTo(user.enrolmentsLink))
+        .willReturn(
+          aResponse()
+              .withStatus(200)
+              .withBody(
+                s"""
+                   |[{"key":"HMRC-AS-AGENT","identifiers":[{"key":"Arn","value":"ARN0001"}],"state":"Activated"}]
```

Comment:
Perhaps we can remove it until we know what the key will be? I'm looking at the code now to see whether it is actually needed.